src/build.rs

```diff
@@ -496,12 +506,22 @@ impl<F: Form, N, T> FieldBuilder<F, N, T> {
             marker: PhantomData,
         }
     }
+
+    #[cfg(feature = "docs")]
+    /// Initialize the documentation of a field (for types in portable form, optional).
+    pub fn docs_portable<I>(mut self, docs: I) -> Self
```

Comment:
Since these methods are called `docs_portable`, strictly speaking they should be in a `impl<N, T> FieldBuilder<PortableForm, N, T>` impl.

Alternatively could rename them to `docs_iter` or something.

Reply:
It's kind of similiar doc `doc_always` which is already taken... But yeah lets switch them into a `PortableForm` impl.